software/SolarAISVN/SolarAISVN.ino

```diff
@@ -25,14 +25,21 @@ uint64_t uS_TO_S_FACTOR = 1000000;  // Conversion factor for micro seconds to se
 uint64_t TIME_TO_SLEEP = 120;
 
 int adcValue = 0;
+int batValue = 0;
```

Comment:
Finally you measure the voltage of the LiPo Battery as well!

Reply:
Thanks. Do you have the ohmic values for your voltage divider?

Reply:
Look at this picture and the two resistors. Not parallel, but they divide 1:1

Value can be checked:
![divider](https://user-images.githubusercontent.com/65865481/99943011-2e9ea480-2da3-11eb-8162-9039464a26ff.jpg)